examples/spl_composites/com.ibm.streamsx.testing.java/src/com/ibm/streamsx/testing/java/TestCustom.java

```diff
@@ -0,0 +1,85 @@
+package com.ibm.streamsx.testing.java;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.ibm.streams.operator.StreamSchema;
+import com.ibm.streams.operator.Type;
+import com.ibm.streamsx.topology.TStream;
+import com.ibm.streamsx.topology.Topology;
+import com.ibm.streamsx.topology.context.StreamsContextFactory;
+import com.ibm.streamsx.topology.spl.SPL;
+import com.ibm.streamsx.topology.spl.SPLStream;
+import com.ibm.streamsx.topology.tester.Condition;
+import com.ibm.streamsx.topology.tester.Tester;
+
+
+/*
+ * This class hows how one can test a SPL composite operator using JUnit
+ * and the Streams Java Application API.
+ */
+public class TestCustom {
+	
+	/*
+	 * Method to create the topology, submit the application based on 
+	 * the specified context and validate data
+	 */
+	private void doTestMultiplyBy(String context) throws Exception {
+		
+		// Create topology to drvie the test
+		Topology topo = new Topology("TestMultiplyBy");
+				
+		// Add the toolkit under test to the toolkit path
+		SPL.addToolkit(topo, new File("../com.ibm.streamsx.testing.spl"));
+		SPL.addToolkit(topo, new File("../com.ibm.streamsx.testing.app"));
+		
+		// Get SPL Schema for test result
+		StreamSchema schema = Type.Factory.getStreamSchema("tuple<int32 result>");
+		
+		// Set up parameters for the composite operator
+		Map<String, Object> params = new HashMap<String, Object>();
+		params.put("factor", 3);
+		
+		// Invoke the test composite
+		SPLStream resultStream = SPL.invokeSource(topo, "com.ibm.streamsx.testing.spl::TestMultiplyBy", params, schema);
+		
+		// Convert the SPL Stream to TStream so we can work with it with the tester
```

Comment:
In this case it's not clear why you are converting the SPL stream to a TStream since you are only checking the count.

Reply:
I was setting up for us to be able to validate the stream content.  I did the count condition because it was easiest to do.